### DISCUSSION TITLE
remove_orphaned_nodes/prune

### DISCUSSION THREAD

**Discussion by dominik-kern** [nschloe/meshio#1285] [Q&A] [answered]
In earlier versions there were the functions ``prune`` and later ``remove_orphaned_nodes`` and something about ``lower_dimensional_entities``.
I miss them, as they are useful, particularly for extracting submeshes.
Do they operate now under new names or were they pruned in version 5?

**Comment by nschloe** [comment, marked as answer]
I took all mesh-manipulation routines out of meshio, and into a soon-to-be-released package.

**Reply by dominik-kern** [reply]
Thank you, I am looking forward to it.

**Reply by HanYang1994** [reply]
Just wondering if this new package came out? Really looking forward to it...
